client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx

```diff
@@ -26,6 +27,21 @@ import {
 	SubscriptionsPortal,
 } from '../subscription-manager-context';
 
+const getIcon = (
```

Comment:
You could turn this function into a component (internal to this file), so you would need to memoize it, just add the tags to the markup. And maybe instead of passing the color, you could pass a boolean prop like `notificationDisabled`, that would be used to define the icon and the color. So the usage could be like this:

`<NotificationStatusIcon iconRef={ tooltip } setTooltipVisible={ setTooltipVisible } notificationDisabled={ delivery_methods.email?.send_comments } />`

Reply:
Very nice! I'll implement it.